docs/markdown/CMake-module.md

```diff
@@ -134,37 +133,37 @@ together with the `options` kwarg.
 This object is returned by the `subproject` function described above
 and supports the following methods:
 
- - `dependency(target)` returns a dependency object for any CMake target. The
-   `include_type` kwarg *(new in 0.56.0)* controls the include type of the
```

Comment:
This file has been reformatted a lot, which makes it harder to review and should probably be in its own commit if you want to do it.

Reply:
Undid all the formatting changes, kept only the actual changes in content